sci-electronics/gazebo/gazebo-2.2.1.ebuild

```diff
@@ -40,13 +40,14 @@ DEPEND="${RDEPEND}
 	doc? ( app-doc/doxygen app-text/ronn )
 "
 
-PATCHES=(
-	"${FILESDIR}/${P}-stdint.patch"
-)
-
-SRCDIR="$(ls ${WORKDIR} | grep osrf-gazebo | tail -n 1)"
+SRCDIR="osrf-gazebo-3b8a07b268a5"
```

Comment:
Do you know why the command doesn't work? I'm at a loss with it... Especially since you `ls` the work directory manually...

I would much prefer the command be fixed so that I won't need to update the ebuilds manually in the future (though later versions of Gazebo seem not to have this problem).